wordpress-mobile/WordPress-iOS
# [Stats Refresh] StatsViewModels: Replace default search engine icon

Refs #11080.

I took an attempt at replacing the search engine icon with a Gridicon as referenced in the issue above.

I figured the view model would be the best place for it, as that's all about transforming the data into the correct format to view. I also opted to switch on the existing icon URL filename instead of the referrer title, as I thought that would change for different locales.

I don't love that it's duplicated in two places, although the rest of the method is already duplicated. If we were to use this in more places, then maybe extracting it somewhere else would make sense. But if you have any suggestions for improvements, or you'd tackle this a different way, let me know!

**Before**

<img width="406" alt="Screenshot 2019-05-10 at 11 00 43" src="https://user-images.githubusercontent.com/4780/57519358-f5165d80-7312-11e9-9a41-1cc0107f5d0e.png">

**After**

<img width="406" alt="Screenshot 2019-05-10 at 10 37 45" src="https://user-images.githubusercontent.com/4780/57519277-cac4a000-7312-11e9-9436-d20d485e27b1.png">

**To test:**

* Build and run, navigate to search referrers in Stats, and check it looks as above.
